tarantool_test.go

```diff
@@ -2731,7 +2731,7 @@ func TestClientRequestObjectsWithPassedCanceledContext(t *testing.T) {
 	req := NewPingRequest().Context(ctx)
 	cancel()
 	resp, err := conn.Do(req).Get()
-	if err.Error() != "context is done" {
+	if !strings.HasPrefix(err.Error(), "context is done") {
```

Comment:
Let's check for a more accurate match here (maybe I suggested the wrong regexp - it is just an idea).
```suggestion
	if !regexp.Match("context is done \(request [0-9]+\)", []byte(err.Error())) {
```

Reply:
Fixed: Thank you for regexp! I used it in fix